wgengine/monitor/monitor_freebsd.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build freebsd
```

Comment:
don't make this a package comment. Add a line after it.

But it's unnecessary given the filename, so just delete this line.